### PR TITLE
Feature/add rm command

### PR DESCRIPTION
**Pull request by takvamborgen**

### CODE REVIEW

File: cmd/ls_test.go [hunk 1]
```diff
@@ -34,23 +50,90 @@ func Test(t *testing.T) {
...
+			rest.ListDatasetElement{
+				Path:      "/foo2/baz",
+				CreatedBy: "Bjørn-André Skaar",
+				CreatedAt: time.Date(3000, 1, 1, 0, 0, 0, 123456000, time.UTC),
+				Type:      "BOUNDED",
+				Valuation: "INTERNAL",
+				State:     "INPUT",
+				Depth:     1,
+			},
+		}, // TODO how to solve this better?
```
Comment (hadrienk) [inline cmd/ls_test.go:123]: Not clear what should be solved better?

File: cmd/rm.go [hunk 2]
```diff
@@ -1,20 +1,113 @@
...
+	defer writer.Flush()
+
+	if debug {
+		noOfFiles := 0
+		var totalSize uint64 = 0
+		for _, datasetVersion := range deleteResponse.DatasetVersion {
+			fmt.Fprintf(writer, "Version: %s\n", datasetVersion.Timestamp)
+			for _, deletedFile := range datasetVersion.DeletedFiles {
+				noOfFiles++
+				totalSize = totalSize + deletedFile.Size
```
Comment (hadrienk) [inline cmd/rm.go:91]: We could compute the total/amount of file inside a type method (same as `IsDataset() bool`)

**Review by hadrienk — APPROVED** [review]
Looks good. Two superficial comments if you have time.

File: cmd/ls_test.go (hunk 1 above)
Comment (takvamborgen) [reply cmd/ls_test.go:123]: Sorry for the imprecise TODO description. I want a better way to define expected output than manually add correct number of whitespaces. Which change according to the length of the fields.

**Pull request merged**